cat-launcher/src/pages/BackupsPage/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from "react";
+
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { useGameVariants } from "@/hooks/useGameVariants";
+import { GameVariant } from "@/generated-types/GameVariant";
+import { useBackups } from "@/hooks/useBackups";
+import { DataTable } from "@/components/DataTable";
+import { columns } from "./columns";
+import { useDeleteBackup } from "@/hooks/useDeleteBackup";
+import { useRestoreBackup } from "@/hooks/useRestoreBackup";
+import { DeleteBackupDialog } from "./DeleteBackupDialog";
+import { RestoreBackupDialog } from "./RestoreBackupDialog";
+import { BackupEntry } from "@/generated-types/BackupEntry";
+
+function BackupsPage() {
+  const { gameVariants, isLoading: gameVariantsLoading } = useGameVariants();
+  const [selectedVariant, setSelectedVariant] = useState<GameVariant | null>(
+    null
+  );
+
+  useEffect(() => {
+    if (gameVariants.length > 0 && !selectedVariant) {
+      setSelectedVariant(gameVariants[0].id);
+    }
+  }, [gameVariants, selectedVariant]);
+
+  const { backups, isLoading: backupsLoading } = useBackups(
+    selectedVariant ?? gameVariants[0]?.id
+  );
+  const { deleteBackup } = useDeleteBackup(
+    selectedVariant ?? gameVariants[0]?.id
+  );
+  const { restoreBackup } = useRestoreBackup(
+    selectedVariant ?? gameVariants[0]?.id
+  );
```

Comment:
In cat-launcher/src/pages/BackupsPage/index.tsx around lines 33 to 41, hooks are
being called with selectedVariant ?? gameVariants[0]?.id which can be undefined
when selectedVariant is null and gameVariants is empty; update usage so hooks
are not invoked with undefined by either (1) passing a guarded variant and
skipping hook calls until a valid id exists (e.g., only call
useBackups/useDeleteBackup/useRestoreBackup when selectedVariant or
gameVariants[0]?.id is truthy) or (2) modify the hooks call to pass the id but
enable them conditionally (use the hooks’ query options like enabled:
Boolean(id)) so no API calls occur when id is undefined.